### ISSUE TITLE
Systax Error after compile for production

### ISSUE THREAD

**Issue by danimasa** [electronjs/electron-compile#58]
I'm having the following error after compile my app for production

![image](https://cloud.githubusercontent.com/assets/6359331/12263619/bf383a34-b917-11e5-8e22-0dd92b52ef24.png)

I'm using the `electron-prebuilt-compile` package and using `electron-compile` command to create a .cache folder.

If I remove the .cache folder everything works

**Comment by paulcbetts** [comment]
Can you share this project?

**Comment by paulcbetts** [comment]
Alternately, can you run this with the `DEBUG` environment variable set and paste the log?

**Comment by danimasa** [comment]
```
Thu, 14 Jan 2016 12:44:16 GMT electron-compile:compile-cache Fetching F:\projetos\bsearch\dist\process\main.js from cache
Thu, 14 Jan 2016 12:44:16 GMT electron-compile:compile-cache Path for BabelCompiler: F:\projetos\bsearch\.cache\8009abd99f8e9c6eddc34994607c6f2c015a5d7e
Thu, 14 Jan 2016 12:44:16 GMT electron-compile:compile-cache Set up with parameters: {"name":"BabelCompiler","version":"6.4.0","options":{"presets":["stage-0","es2015"],"sourceMaps":"inline"}}
App threw an error when running [SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode]
```

Here the logs.

**Comment by paulcbetts** [comment]
This sounds as if you don't have a .babelrc / .compilerc and got Babel's default "pass through everything"?

**Comment by stefanbuck** [comment]
Hey, I run into a similar problem with an ES6 application. When I run the application with `electron .` and no `.cache` folder is present, the app boots and all the ES6 and LESS stuff is compiled. If I run `electron-compile --appDir . ./src/ ./static/` to build cache, I get this output in the console.

```
Using NODE_ENV = development
Failed to compile file: src/renderer/components/compatibility-warning.jsx
src/renderer/components/compatibility-warning.jsx: Unexpected token (32:6)
...
Failed to compile file: src/renderer/main.js
src/renderer/main.js: Unexpected token (26:15)
```

When I start the app again with `electron .` a dialog pops up with this message:

```
Uncaught Exception:
SyntaxError: Unexpected token import
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:404:25)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/stefanbuck/projects/yeoman-app/node_modules/electron-compile/lib/require-hook.js:35:14)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
    at Module.require (module.js:366:17)
    at Object.init (/Users/stefanbuck/projects/yeoman-app/node_modules/electron-compile/lib/config-parser.js:136:16)
    at EventEmitter.<anonymous> (/Users/stefanbuck/projects/yeoman-app/src/browser/main.js:8:31)
    at emitOne (events.js:82:20)
    at EventEmitter.emit (events.js:169:7)
```

It looks like the code was not transpiled via babel. Did you have any ideas what could cause this issue? Did I configured something wrong? Before it was working with `electron-compilers@1.x`. The code is available here https://github.com/stefanbuck/yeoman-app/tree/fix-build

Thanks for your time

**Comment by paulcbetts** [comment]
@stefanbuck I'll have a look, thanks for the great repo steps

**Comment by jronallo** [comment]
After changing something that caused a similar SyntaxError to come up, I blew away my node_modules, reinstalled everything and it worked again without changing anything else. Don't know why that might be or why that might of worked.

**Comment by stefanbuck** [comment]
@paulcbetts did you had time to look into this yet?

**Comment by paulcbetts** [comment]
@stefanbuck Hm, something odd's going on with how we try to fix up plugins, I'll try to fix this up. In the meantime, you should add a `.compilerc` to your project:

```
{
  "application/javascript": {
    "presets": ["stage-0", "es2015", "react"]
  }
}
```

**Comment by paulcbetts** [comment]
@stefanbuck Bug fixed, update to 2.0.5 of `electron-compilers` and you should be all set. Thanks all for reporting this!

**Comment by stefanbuck** [comment]
@paulcbetts thanks a lot for the fix :+1:
